# References

## [1] Hamelin et al., 2025
Predicting pathogen evolution and immune evasion in the age of artificial intelligence. *Computational and Structural Biotechnology Journal*, 23, Article 1044. [https://doi.org/10.1016/j.csbj.2025.03.044](https://doi.org/10.1016/j.csbj.2025.03.044)

## [2] Vaswani et al., 2017
Attention is all you need. In *Proceedings of the 31st International Conference on Neural Information Processing Systems (NIPS ‘17)*, Curran Associates Inc., Red Hook, NY, USA, 6000-6010. [https://doi.org/10.48550/arXiv.1706.03762](https://doi.org/10.48550/arXiv.1706.03762)

## [3] Asgari & Mofrad, 2015
Continuous distributed representation of biological sequences for deep proteomics and genomics. *PLoS ONE*, 10(11), e0141287. [https://doi.org/10.1371/journal.pone.0141287](https://doi.org/10.1371/journal.pone.0141287)

## [4] Nambiar et al., 2020
Transforming the language of life: Transformer neural networks for protein prediction tasks. *Proceedings of the 11th ACM International Conference on Bioinformatics, Computational Biology and Health Informatics (BCB '20)*, Article 5, 1–8. [https://doi.org/10.1145/3388440.3412467](https://doi.org/10.1145/3388440.3412467)

## [5] Penić et al., 2025
RiNALMo: general-purpose RNA language models can generalize well on structure prediction tasks. *Nature Communications*, 16(1), Article 60872. [https://doi.org/10.1038/s41467-025-60872-5](https://doi.org/10.1038/s41467-025-60872-5)

